delivery/repositories.py

```diff
@@ -0,0 +1,48 @@
+from abc import ABC, abstractmethod
+from delivery.error_handling import WrongIdError
+from delivery.models import Restaurant
+
+
+class AbstractRestaurantRepository(ABC):
+    @abstractmethod
+    def create(self, content: Restaurant):
```

Comment:
use typing for both arguments and return value: 
```
from typing import List

...
def create(self, content: Restaurant) -> None
...

def get_all(self) -> List[Restaurant] # or Iterable[Restaurant]

```

Reply:
good